### PR TITLE
Fix shared gallery navigation

### PR DESCRIPTION
**Pull request by michag86**
Fixes:  #286

### CODE REVIEW

**Comment by michag86** [comment]
If this fix is okay, I will create a backport for stable12

File: js/slideshowcontrols.js [hunk 1]
```diff
@@ -350,7 +350,9 @@
 		 * @private
 		 */
 		_next: function () {
-			Gallery.Share.hideDropDown();
+			if(!_.isUndefined(Gallery.Share)){
```
Comment (icewind1991) [inline js/slideshowcontrols.js]: You can just use `if(Gallery.Share)`

Comment (michag86) [inline js/slideshowcontrols.js]: I've just copied it from here... https://github.com/nextcloud/gallery/blob/41400d13581407675255ac2a172a0aa73eba1b0d/js/gallery.js#L340

**Comment by oparoz** [comment]
@michag86 Please backport :)

**Pull request merged**